src/improvements/tasks/MultisigTask.sol

```diff
@@ -439,6 +476,9 @@ abstract contract MultisigTask is Test, Script, ITask {
         for (uint256 i = 0; i < chains.length; i++) {
             _validate(chains[i].chainId);
         }
+
+        /// reverts by default and must be overridden by the child templates
```

Comment:
Is this true? I see `checkStateDiff` in this function that is implemented and doesn't seem to revert.